DOCS-3290: import accounting (#2173)
MultiSafepay/docs: content/accounting/accounting-integrations.md

---
title: "Accounting integrations"
category: 62962dee7af1c800355771a1
order: 200
hidden: false
---

> This section contains integrations that can be used with your accounting platform.

Reviewer: ```suggestion
This section contains integrations you can use with your accounting platform.
```
